api/migrations/versions/0199_change_provider_details.py

```diff
@@ -0,0 +1,36 @@
+"""
+
+Revision ID: 0199
+Revises: 0198
+Create Date: 2018-09-10 09:34:40.184871
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0199'
+down_revision = '0198'
+
+
+def upgrade():
+    op.execute("UPDATE provider_details SET supports_international=True WHERE identifier='twilio'")
```

Comment:
our current twilio number can only send to domestic numbers. we should look at purchasing one that supports international; or creating another "twilio-international" provider that could be used here

Reply:
peter and I checked and the current twilio number can send to international numbers (except for USA/Canda which is +1 prefix)